src/openforms/emails/models.py

```diff
@@ -65,5 +65,5 @@ def __str__(self):
         return _("Confirmation email template - {form}").format(form=form_str)
 
     @property
-    def is_usable(self) -> bool:
-        return bool(self.subject and self.content)
+    def is_empty(self) -> bool:
```

Comment:
Im not sure if `is_empty` makes more sense, since it concerns 2 properties of `ConfirmationEmailTemplate`. I think I would leave `is_usable` and maybe add a docstring saying that the `ConfirmationEmailTemplate` is only usable if both the subject / content are present?